src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -63,10 +64,18 @@ std::string Notification::toJson()
 {
   JsonHelper jh;
 
-  jh.addString("callback", this->callback);
-  if (!this->throttling.isEmpty())
-  {    
-    jh.addRaw("throttling", "\"" + this->throttling.get() + "\"");
+  jh.addString("callback", this->callback);  
+  if (this->throttling > 0)
+  {
+    jh.addLongLong("throttling", this->throttling);
+  }
+  if (this->count > 0)
+  {
+    jh.addLongLong("count", this->count);
+  }
+  if (this->lastNotification > 0)
+  {
+    jh.addDate("lastNotification", this->lastNotification);
   }
```

Comment:
Actually, this is a pretty good candidate for the one-liner ifs (if you want to save lines and improve readibility). Look:

    if (this->throttling > 0)           { jh.addLongLong("throttling", this->throttling);         }
    if (this->count > 0)                { jh.addLongLong("count", this->count);                   }
    if (this->lastNotification > 0)     { jh.addDate("lastNotification", this->lastNotification); }

Just a suggestion.